article/app/views/liveblog/liveBlogBlocks.scala.html

```diff
@@ -2,14 +2,24 @@
 @import model.Article
 @import model.liveblog.{LiveBlogDate,BodyBlock}
 @import views.BodyCleaner
-@import conf.switches.Switches.LiveUpdateAmpSwitch
+@import conf.switches.Switches.{LiveUpdateAmpSwitch,AmpLiveBlogNewsArticleSwitch}
 
 @(blocks: Seq[BodyBlock], article: Article, timezone: DateTimeZone, amp: Boolean = false)(implicit request: RequestHeader)
 
 @import common.LinkTo
 
+@* Aug 2016: Google only supports NewsArticle in their amp carousel. To remove once Google starts supporting other schemas like schema.org/LiveBlogPosting *@
+@isNewsArticle() = @{
```

Comment:
You're right, it's not a very clear name, but there is a lot of context to convey.

Until Google supports the `LiveBlogPosting` article schema in their fancy AMP carousel in search results, AMP live blogs follow the `NewsArticle` schema. This function determines whether we are building the AMP version of the live blog, and whether it has been structured using the `NewsArticle` schema.

Can you think of a name that might better convey this intention?